src/Confluent.Kafka/Consumer.cs

```diff
@@ -751,6 +751,16 @@ private void LogCallback(IntPtr rk, int level, string fac, string buf)
         /// </remarks>
         public Consumer(IEnumerable<KeyValuePair<string, object>> config)
         {
+            var librdkafkaPath = config.FirstOrDefault(prop => prop.Key == "librdkafka.path").Value;
+            config = config.Where(prop => prop.Key != "librdkafka.path");
+
+            if (LibRdKafka.IsInitialized && librdkafkaPath != null)
```

Comment:
there is actually a possible benign race condition here that will never matter in practice.